pkg/release/release.go

```diff
@@ -116,6 +116,13 @@ const (
 	// BuildDir is the default build output directory.
 	BuildDir = "_output"
 
+	// WorkspaceDir is the global directory where the stage and release process
+	// happens.
+	WorkspaceDir = "/workspace"
+
+	// gitRoot is the local repository root of k/k.
+	GitRoot = WorkspaceDir + "/src/k8s.io/kubernetes"
+
```

Comment:
I'm not a fan of re-defining those here again, let's pass the `GitRoot` used in the functions in `pkg/release/workspace.go` as argument.

Reply:
I wanted to have the functions as easy as possible in order to avoid rethinking the paths when using them. But now that I changed them, it turns out that passing the git root is better as I can write a test.

I plan to replace them once we have the supported platforms code bit for now they will take the gitRoot as arg.

OK, done